cartoframes/data/observatory/catalog/repository/repo_client.py

```diff
@@ -11,7 +12,7 @@ class RepoClient:
 
     def __init__(self):
         self._do_dataset = None
-        default_credentials = Credentials(DEFAULT_USER)
+        default_credentials = utils.get_credentials() or Credentials(DEFAULT_USER)
```

Comment:
This change LGTM. Using the default credentials, if set will increase a bit the presence of users in the catalog metrics.

Reply:
🙇  It also allow calls like `Catalog().categories` to use a different `base_url` instead of using always `https:\\{}.carto.com`